pkg/mdformatter/linktransformer/link.go

```diff
@@ -207,32 +207,48 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 		}
 	}
 
+	linktransformerMetrics := newLinktransformerMetrics(reg)
+	collector := colly.NewCollector(colly.Async(), colly.StdlibContext(ctx), colly.MaxDepth(1))
+	collector.SetClient(&http.Client{
+		Timeout: 30 * time.Second,
```

Comment:
mdox already has a [parameter](https://github.com/bwplotka/mdox/blob/main/pkg/mdformatter/linktransformer/link.go#L240) in config which allows users to customize this, as different users will likely have different types of docs (how many links/type of links). 🙂 

We use this [parameter](https://github.com/prometheus-operator/prometheus-operator/blob/main/.mdox.validate.yaml#L2) in prometheus-operator too.

Reply:
Fixed in https://github.com/bwplotka/mdox/pull/86/commits/a02848d8d61d5cacc7455c3960df4a90387aa628